dashboard/app/controllers/api/v1/pd/workshop_attendance_controller.rb

```diff
@@ -29,7 +29,7 @@ def update
       attending_user_ids = []
       supplied_attendances.each do |attendance|
         if attendance[:id]
-          attending_user_ids << attendance[:id]
+          attending_user_ids << attendance[:id].to_i
```

Comment:
Yes, I uncovered this bug while working on this change. The supplied params are string, but it didn't cause any issues before the idempotence case.